drafts/use-cases/1.entry-point.md

```diff
@@ -0,0 +1,108 @@
+## Entry point
+
+### Story
+As an application user
+I want my application to consume an entry point for the Web API by providing a base URL
+So I can start managing my events.
+
+### Usage
+```javascript
+var client = new HydraClient();
+var data = client.get("/api");
+for (var control of data.controls) {
+    // do something with the _control_
+}
+```
+
+### Details
+We can imaging a situation, when an application has a base URL provided, i.e. in a configuration or in the input field.
+Once provided, I'd expect the application to connect to the entry-point of the Web API and download initial
+details on what else can be done with that API from that very point.
+
+```
+GET /api
+
+HTTP 200 OK
+```
+```javascript
+{
+    "@id": "/api",
+    "operations": [
+        {
+            "@id": "/api/events",
+            "title": "Events",
+            "httpMethod": "GET",
+            "returns": "hydra:Collection"
+        },
+        {
+            "@id": "/api/events",
+            "title": "Create new event",
+            "httpMethod": "POST",
+            "expects": "schema:Event"
+        }
+    ]
+}
+```
+
+With this details, client application is aware of two possible operations that can be made from that point:
+
+- GET events
+- POST an event
+
+We can imagine a UI application with some fancy logic that would takes all
+GET operations returning _hydra:Collection_ and displays them in a main menu.
+
+### Considerations
+
+#### JSON array root security issues
```

Comment:
This is a good point but I would move it to a separate document which collects security considerations.